udev/CMakeLists.txt

```diff
@@ -1,3 +1,7 @@
 install(FILES 50-revpi.rules
-  DESTINATION /lib/udev/rules.d
+  DESTINATION ${CMAKE_INSTALL_LIBDIR}/udev/rules.d
+)
+
+install(PROGRAMS revpi_mac
+  DESTINATION ${CMAKE_INSTALL_LIBDIR}/udev
```

Comment:
This will result in `/usr/lib/aarch64-linux-gnu/udev/revpi_mac`.

`${CMAKE_INSTALL_LIBDIR}/../udev/rules.d` will work, build a test package with that.